xml/vk.xml

```diff
@@ -8776,7 +8776,7 @@ typedef void* <name>MTLSharedEvent_id</name>;
         <type category="struct" name="VkLatencySurfaceCapabilitiesNV" structextends="VkSurfaceCapabilities2KHR">
             <member values="VK_STRUCTURE_TYPE_LATENCY_SURFACE_CAPABILITIES_NV"><type>VkStructureType</type> <name>sType</name></member>
             <member optional="true">const <type>void</type>*                              <name>pNext</name></member>
-            <member optional="false,true"><type>uint32_t</type>                           <name>presentModeCount</name></member>
+            <member optional="true"><type>uint32_t</type>                                 <name>presentModeCount</name></member>
             <member optional="true" len="presentModeCount"><type>VkPresentModeKHR</type>* <name>pPresentModes</name></member>
```

Comment:
These feel like they should not have `optional` attributes at all, but should be marked `returnedonly`.

Reply:
The user can write to `presentModeCount` to get a reduced subset of optimized present modes so `returnedonly` doesn't seem to apply here:
```
If pname:pPresentModes is `NULL`, then the number of present modes that are
optimized for use with low latency mode returned in pname:presentModeCount.
Otherwise, pname:presentModeCount must be set by the user to the number of
elements in the pname:pPresentModes array, and on return the variable is
overwritten with the number of values actually written to
pname:pPresentModes.
If the value of pname:presentModeCount is less than the number of optimized
present modes, at most pname:presentModeCount values will be written to
pname:pPresentModes.
```